x-pack/test/functional/apps/visualize/precalculated_histogram.ts

```diff
@@ -24,37 +24,64 @@ export default function({ getService, getPageObjects }: FtrProviderContext) {
       return esArchiver.unload('pre_calculated_histogram');
     });
 
-    const initHistogramBarChart = async () => {
-      await PageObjects.visualize.navigateToNewVisualization();
-      await PageObjects.visualize.clickVerticalBarChart();
-      await PageObjects.visualize.clickNewSearch('histogram-test');
-      await PageObjects.visChart.waitForVisualization();
-    };
-
-    const getFieldOptionsForAggregation = async (aggregation: string): Promise<string[]> => {
-      await PageObjects.visEditor.clickBucket('Y-axis', 'metrics');
-      await PageObjects.visEditor.selectAggregation(aggregation, 'metrics');
-      const fieldValues = await PageObjects.visEditor.getField();
-      return fieldValues;
-    };
-
     it('appears correctly in discover', async function() {
       await PageObjects.common.navigateToApp('discover');
       const rowData = await PageObjects.discover.getDocTableIndex(1);
       expect(rowData.includes('"values": [ 0.3, 1, 3, 4.2, 4.8 ]')).to.be.ok();
     });
 
-    it('appears in the field options of a Percentiles aggregation', async function() {
-      await initHistogramBarChart();
-      const fieldValues: string[] = await getFieldOptionsForAggregation('Percentiles');
-      log.debug('Percentiles Fields = ' + fieldValues);
-      expect(fieldValues[0]).to.be('histogram-content');
-    });
+    describe('works in visualizations', () => {
+      before(async () => {
+        await PageObjects.visualize.navigateToNewVisualization();
+        await PageObjects.visualize.clickDataTable();
+        await PageObjects.visualize.clickNewSearch('histogram-test');
+        await PageObjects.visChart.waitForVisualization();
```

Comment:
Seems to be an extra wait, should work fine without it since the `clickNewSearch` is waiting for `header.waitUntilLoadingHasFinished();`

Reply:
Better save then sorry ;)